.github/workflows/update-latest-tag.yml

```
@@ -0,0 +1,62 @@
+  workflow_dispatch:
+    inputs:
+      version_tag:
+        description: 'Version tag to point latest to (e.g., v2.0.5)'
+        required: true
+        type: string
+
+jobs:
+  update-latest:
+    name: Update Latest Tag
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)